src/server/scripts/Commands/cs_debug.cpp

```diff
@@ -1061,23 +991,25 @@ class debug_commandscript : public CommandScript
         return true;
     }
 
-    static bool HandleDebugEnterVehicleCommand(ChatHandler* handler, char const* args)
+    static bool HandleDebugEnterVehicleCommand(ChatHandler* handler, uint32 entryOrSeatId, Optional<int8> seatIdArg)
     {
         Unit* target = handler->getSelectedUnit();
         if (!target || !target->IsVehicle())
             return false;
 
-        if (!args)
-            return false;
-
-        char* i = strtok((char*)args, " ");
-        if (!i)
-            return false;
-
-        char* j = strtok(nullptr, " ");
+        int8 seatId = -1;
+        uint32 entry = 0;
 
-        uint32 entry = (uint32)atoi(i);
-        int8 seatId = j ? (int8)atoi(j) : -1;
+        if (seatIdArg)
+        {
+            entry = entryOrSeatId;
+            seatId = *seatIdArg;
+        }
+        else
+        {
+            entryOrSeatId = 0;
+            seatId = entryOrSeatId;
+        }
 
```

Comment:
Is this a change of functionality? As I see it, before it was always `entry seatId`, and now it is `[entry] seatId`?

PS: This entire check looks screwy to me. Consider making `entry` optional instead of what you currently do.